pkgs/by-name/bp/bpftop/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, elfutils
+, zlib
+, libbpf
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bpftop";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "Netflix";
+    repo = "bpftop";
+    rev = "v${version}";
+    hash = "sha256-HP8ubzCfBNgISrAyLACylH4PHxLhJPzIQFmIWEL5gjo=";
+  };
+
+  cargoHash = "sha256-+zh7GZ/fbhxLNQkkHFZqtJxy2IeS+KX5s2Qi5N21u/0=";
+
+  buildInputs = [
+    elfutils
+    libbpf
+    zlib
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  meta = {
+    description = "Bpftop provides a dynamic real-time view of running eBPF programs";
```

Comment:
```suggestion
    description = "A dynamic real-time view of running eBPF programs";
```

Reply:
Sorry I missed this the first time around - the description shouldn't refer to the package name (see [docs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes)).

Good to merge once this is addressed.